proto/marshal/compat.py

```diff
@@ -55,6 +49,25 @@
         pass
 
 
+
+try:
+    from google.protobuf.pyext import _message as _message_pyext
+except ImportError:
+    _message_pyext = None
```

Comment:
Move this import statement to the top of the file since we're importing it as `_message_pyext`